android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -442,28 +439,6 @@ public void onTrimMemory(int level)
 		super.onTrimMemory(level);
 	}
 
-	public void postAppInfo()
-	{
-		deployData = new TiDeployData(this);
-
-		String deployType = this.appProperties.getString("ti.deploytype", "unknown");
-		if ("unknown".equals(deployType)) {
-			deployType = this.appInfo.getDeployType();
-		}
-
-		String buildType = this.appInfo.getBuildType();
-		if (buildType != null && !buildType.equals("")) {
-			APSAnalyticsMeta.setBuildType(buildType);
-		}
-
-		APSAnalyticsMeta.setAppId(this.appInfo.getId());
-		APSAnalyticsMeta.setAppName(this.appInfo.getName());
-		APSAnalyticsMeta.setAppVersion(this.appInfo.getVersion());
-		APSAnalyticsMeta.setDeployType(deployType);
-		APSAnalyticsMeta.setSdkVersion(getTiBuildVersion());
-		APSAnalytics.getInstance().setMachineId(this);
```

Comment:
can you check if `Ti.Platform.id` is still working. That is returning `APSAnalytics.getInstance().getMachineId()` and it looks like this is the place it gets set.